lib/reporters/mini.js

```diff
@@ -77,6 +78,10 @@ MiniReporter.prototype.prefix = function (str) {
 MiniReporter.prototype._test = function (test) {
 	var title = test.title;
 
+	if (title.length > process.stdout.columns) {
```

Comment:
I used to not have the check but then for some reason the mini tests were [failing](http://i.imgur.com/zHqqEW9.png). Didn't have time to diagnose why though.